guides/common/modules/proc_installing-and-configuring-puppet-agent-manually.adoc

```diff
@@ -48,44 +42,37 @@ ifndef::satellite[]
 # zypper install puppet-agent
 ----
 endif::[]
-. Append the following server and environment settings to the `/etc/puppetlabs/puppet/puppet.conf` file.
+. Configure the Puppet agent.
 Set the `environment` parameter to the name of the Puppet environment to which the host belongs:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-environment = _My_Puppet_Environment_
-server = _{foreman-example-com}_
+# puppet config set server _{foreman-example-com}_ --section agent
```

Comment:
Not sure where to note this, but we should warn the user that it may be needed to load the settings.

The `puppet-agent` package installs `/etc/profile.d/puppet-agent.sh` which modifies `PATH` and `MANPATH` env vars. This is executed when your shell starts up, but you need to load it after running the `$x install puppet-agent` package if you're in the same shell.

The easiest way is:
```sh
. /etc/profile.d/puppet-agent.sh
```

Otherwise you end up with "command not found" errors.

This is a one time thing, but if a user just follows the procedure it's a common error. I'm unsure about the convention of documenting it, but please consider it.

Reply:
I simply added a step right after the package installation.